eZ/Publish/API/Repository/Tests/FieldType/EmailAddressIntegrationTest.php

```diff
@@ -363,4 +363,14 @@ protected function getSearchTargetValueTwo()
         // ensure case-insensitivity
         return strtoupper($this->getValidSearchValueTwo());
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFullTextIndexedFieldData()
+    {
+        return [
+            ['holmes4', 'wyoming'],
```

Comment:
So here's the thing: if I do a FullText search for the entire e-mail, e.g. `holmes4@ez.no`, I get all Content items containing parts of e-mail: `holmes4` or `ez` or `no`. It looks like parser treats this as separate words.

My expectation in this case would be to find one Content item with field matching exactly the given e-mail.
However Solr FullText search gets me the same result, so maybe my expectations are wrong?

Feedback ping @andrerom @m-tyrala (as you're the one who found this).